tests/System.Text.Utf8String.Tests/Utf8String.RandomTests.cs

```diff
@@ -132,32 +132,6 @@ public unsafe void StringEqualsConvertedToUtf16String(bool expected, string str1
         };
 
         [Theory]
-        [InlineData("", 'a')]
-        [InlineData("abc", 'a')]
-        [InlineData("v", 'v')]
-        [InlineData("1", 'a')]
-        [InlineData("1a", 'a')]
-        public void StartsWithCodeUnit(string s, char c)
```

Comment:
Should we substitute these Utf8String tests that are being removed with tests on the new Utf8Span APIs?

Reply:
THis test is removed because it tests code-unit based APIs, which I removed.